data/css/modules/card/_media.scss

```diff
@@ -2,23 +2,40 @@
     background-color: white;
 
     &__caption {
-        font-size: 20px;
+        font-size: 16px;
         font-family: $body-font;
-        color: #666666;
-        padding: 20px 20px 0px 20px;
+        color: transparentize(black, 1 - 0.60);
     }
 
     &__attributionText {
         font-weight: bold;
-        font-size: 16px;
-        font-family: $display-font;
+        font-size: 14px;
+        font-family: $support-font;
         color: $accent-light-color;
-        padding: 10px 20px 20px 20px;
 
         &:hover {
             color: $accent-dark-color;
         }
     }
+
+    &__caption,
+    &__attributionText {
+        padding: 25px;
+    }
+}
+
+// When both the caption & attribution are present in the lightbox,
+// edit padding to accommodate both
+.CardMedia--caption.CardMedia--attribution {
+    .CardMedia {
```

Comment:
So what's happening (and what we discussed earlier on this thread, tell me if I interpreted your suggestion wrongly) is this:
- The `Card.Media` module has a class called `.CardMedia`, and when both attribution text & caption text are present, that module is given 2 new classes called `.CardMedia--attribution` and `.CardMedia--caption` in the JS file.
- This CSS is saying that whenever the outer `Card.Media` module has both of these classes applied to it, it should style its inner widgets (attribution text, with class `.CardMedia__attributionText`, and caption text, with class `.CardMedia__caption`) with different paddings than they usually have.

This works because the attribution text & the caption text are inside of the `Card.Media` module.